base_layer/wallet/src/output_manager_service/handle.rs

```diff
@@ -108,6 +109,23 @@ impl OutputManagerHandle {
         }
     }
 
+    pub async fn get_coinbase_spending_key(
+        &mut self,
+        tx_id: u64,
+        amount: MicroTari,
+        maturity_height: u64,
```

Comment:
The wallet does not have access to the ConsensusConstants struct right? Because that value is consensus and is stored in there.

Reply:
Yup, Wallet only has the Transaction crate as a dependency because Core has some dependencies that cannot be built for ARM.